CHANGELOG.md

```diff
@@ -1,10 +1,28 @@
+4.54.0
+==========
+### New Features
+- Acquiring tokens with managed identity is now generally available. See [4125](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4125) and [documentation for managed identity in MSAL.NET](https://aka.ms/msal-net-managed-identity).  
+- Updated the managed identity API to specify the identity type when creating an `ManagedIdentityApplication`. See [4114](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4114).  
+- When acquiring tokens with managed identity and using the default HTTP client, MSAL will retry the request for certain exception codes. See [4067](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4067).  
+- `MsalManagedIdentityException` now includes the Azure source used. See [4041](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4041).  
```

Comment:
think this is new for GA `MsalManagedIdentityException`, if so we may need to elaborate on the new exception type. And also we cay what it will contain